lib/logger_json/formatters/google_cloud_logger.ex

```diff
@@ -2,6 +2,7 @@ defmodule LoggerJSON.Formatters.GoogleCloudLogger do
   @moduledoc """
   Google Cloud Logger formatter.
   """
+
```

Comment:
We don't add new line between moduledoc and `alias/import/include` to make the header smaller.